.travis.yml

```diff
@@ -1,29 +1,71 @@
 language: cpp
-os:
-- linux
-- osx
-compiler:
-- clang
-- gcc
+
+matrix:
+  include:
+     # Coverage
+     - os: osx
+       compiler: clang
+       env: NODE_VERSION="0.10" COVERAGE=true
+     # Linux
+     - os: linux
+       compiler: clang
+       env: NODE_VERSION="0.10"
+     - os: linux
+       compiler: clang
+       env: NODE_VERSION="0.12"
+     # OS X
+     - os: osx
+       compiler: clang
+       env: NODE_VERSION="0.10"
+     - os: osx
+       compiler: clang
+       env: NODE_VERSION="0.12"
+
 env:
-  matrix:
-  - NODE_NVM_VERSION="0.8.26"
-  - NODE_NVM_VERSION="0.10"
   global:
-  - BUILD: '/tmp/fontnik-build'
-  - PKG_CONFIG_PATH: '/tmp/fontnik-build/lib/pkgconfig'
-  - secure: "XV0lekmfgT+D9t0ZTIU+UJF6g+p3cBQMO6T6C9lkoKTC0YbtLtxSFtBahD/4PjL86DMJgTaf1nBmxqOxbrfkcpJUxnLe3r8u4Z2L/+7+QSACLNktlIfWNSO+33WxKNb4mVw6jMFZIo4ZurF016MXYzLzjpxRELW2oO2STUs2m44="
-  - secure: "CQNHbxw8yHlAdUVbKokHzHmj7C+duXP3mifWOkZm9GKw4myWsRFhhoSYZmOSkgj9EWfYYkedrqEr9+GaMg9rkVJuO/7jzn6S+M7CFXKJju6MoZEDO6WcFva4M8pw6IFb9q22GcQ+OsE8/i0DwchTokyFkNb3fpwWuwROUPQ/nWg="
+   - JOBS: "8"
+   - BUILD: '/tmp/fontnik-build'
+   - PKG_CONFIG_PATH: '/tmp/fontnik-build/lib/pkgconfig'
+   - secure: "XV0lekmfgT+D9t0ZTIU+UJF6g+p3cBQMO6T6C9lkoKTC0YbtLtxSFtBahD/4PjL86DMJgTaf1nBmxqOxbrfkcpJUxnLe3r8u4Z2L/+7+QSACLNktlIfWNSO+33WxKNb4mVw6jMFZIo4ZurF016MXYzLzjpxRELW2oO2STUs2m44="
+   - secure: "CQNHbxw8yHlAdUVbKokHzHmj7C+duXP3mifWOkZm9GKw4myWsRFhhoSYZmOSkgj9EWfYYkedrqEr9+GaMg9rkVJuO/7jzn6S+M7CFXKJju6MoZEDO6WcFva4M8pw6IFb9q22GcQ+OsE8/i0DwchTokyFkNb3fpwWuwROUPQ/nWg="
+
 before_install:
-- export platform=$(uname -s | sed "y/ABCDEFGHIJKLMNOPQRSTUVWXYZ/abcdefghijklmnopqrstuvwxyz/")
-- export PATH="$BUILD/bin:$PATH"
-- if [[ "$platform" == "linux" ]]; then export CXX=g++-4.8; export CC=gcc-4.8; fi
-- ./deps/nvm_install.sh
-- ./deps/travis_build.sh
+ - export COVERAGE=${COVERAGE:-false}
+ - export platform=$(uname -s | sed "y/ABCDEFGHIJKLMNOPQRSTUVWXYZ/abcdefghijklmnopqrstuvwxyz/")
```

Comment:
I think we can get rid of this now that multi-OS builds are enabled and just use `${TRAVIS_OS_NAME}` instead?